specification/elasticsan/ElasticSan.Management/back-compatible.tsp

```diff
@@ -0,0 +1,89 @@
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.ElasticSan;
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ElasticSanUpdate.properties);
```

Comment:
are we going to do better justifications for all these FIXMEs as part of the conversion? Or is the plan to fix them during future iterations before PR is completed?

Reply:
The plan is to fix them after the PR is completed.